cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewManager.java

```diff
@@ -319,9 +321,13 @@ protected void configure() {
         protected void configure() {
           bind(AccessControllerInstantiator.class).toInstance(accessControllerInstantiator);
```

Comment:
Do we need to bind the AccessControllerInstantiator?

Reply:
That's existing code, I assume the binding is correct